src/pages/newAquarium/index.tsx

```diff
@@ -0,0 +1,123 @@
+import { Col, Container, Row } from 'react-bootstrap';
+import Form from 'react-bootstrap/Form';
+import { useTranslation } from 'react-i18next';
+
+import { Button, Card, Input, Tile } from '@/components';
+
+export const NewAquariumView: React.FC = () => {
+  const { t } = useTranslation();
+  return (
+    <Form>
+      <h3 className="mb-4">{t('aquarium_setup')}</h3>
```

Comment:
IMO tutaj najlepiej byłoby wydzielić osobny komponent np. `Title`, który przyjmie title jako props i go wyświetli. Zakładam, że każdy ekran ma jakiś tytuł i na każdym powinno się tak samo to wyświetlać.

Reply:
Spoko, zrobiłem komponent